packages/react-conformance/src/utils/getCallbackTypeNode.ts

```diff
@@ -0,0 +1,104 @@
+import * as ts from 'typescript';
```

Comment:
This is copy-pasted from the deleted part in getCallbackArguments. I don't understand it enough to make any change :/